src/benchmarks/micro/corefx/System.Xml.Linq/Perf.XName.cs

```diff
@@ -0,0 +1,31 @@
+﻿using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Xml.Linq
+{
+    [BenchmarkCategory(Categories.CoreFX)]
+    public class Perf_XName
+    {
+        private XName _noNamespace;
+        private XName _hasNamespace;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            _noNamespace = XName.Get("Root");
+            _hasNamespace = XName.Get("{http://www.example.test}Root");
+        }
+
+        [Benchmark]
+        public XName GetLocalName() => _noNamespace.LocalName;
+
+        [Benchmark]
+        public string GetEmptyNameSpaceName() => _noNamespace.NamespaceName;
+
+        [Benchmark]
+        public XName GetLocalNameFromExpandedName() => _hasNamespace.LocalName;
+
+        [Benchmark]
+        public string GetNonEmptyNameSpaceName() => _hasNamespace.NamespaceName;
+    }
```

Comment:
which of these properties perform some actual work (not just returning a field value)?

EDIT: I've run these benchmarks and I can see that two of them just return a field value and report `0ns` as a result.

|                       Method |       Mean |
|----------------------------- |-----------:|
|                 GetLocalName | 29.4854 ns |
|        GetEmptyNameSpaceName |  0.0000 ns |
| GetLocalNameFromExpandedName | 33.4970 ns |
|     GetNonEmptyNameSpaceName |  0.0000 ns |

Please remove `GetEmptyNameSpaceName` and `GetNonEmptyNameSpaceName`

Reply:
I see, seems better to check ToString() instead